src/Cake.Core/CakeReportPrinter.cs

```diff
@@ -30,13 +30,18 @@ public CakeReportPrinter(IConsole console, ICakeContext context)
         }
 
         /// <inheritdoc/>
-        public void Write(CakeReport report)
+        public void Write(CakeReport report, Verbosity verbosity)
         {
             if (report == null)
             {
                 throw new ArgumentNullException(nameof(report));
             }
 
+            if (verbosity <= Verbosity.Quiet)
+            {
+                return;
+            }
+
```

Comment:
Thanks for the PR @deniszykov! As you can see in [this comment in the original issue](https://github.com/cake-build/cake/issues/2101#issuecomment-376160140) we wouldn't want to change the semantics of the `Quiet` verbosity in order to hide the report - this means users don't have a way to suppress the log messages and see the report anymore.

Could you add a new command-line argument to Cake called `--no-report` that hides the report? (regardless of the Verbosity)